_inc/client/README.md

```diff
@@ -61,14 +61,24 @@ Every action creator defined in the **Admin Page** returns a Promise and is buil
 
 ### Internationalization of the Admin Page
 
-The **Admin Page** takes advantage of [i18n-calypso]() for internationalization purposes.
-
-Internally we use the `translate` function exported by `i18n-calypso` by aliasing to `__()`:
-
-```
-import { translate as __ } from 'i18n-calypso';
-...
-<div> { __( 'String' ) } </div>
+The **Admin Page** takes advantage of two core WordPress packages for internationalization purposes:
+
+- [`@wordpress/i18n`](https://www.npmjs.com/package/@wordpress/i18n) gives us access to internationalization functions we're used to in WordPress: `__n`, `_n`, `_x`, `_nx`, and `sprintf`.
+- [`@wordpress/element`](https://www.npmjs.com/package/@wordpress/element) and its `createInterpolateElement` function allow us to use React elements in strings. `createInterpolateElement` is available in WordPress 5.5 and in latest versions of the Gutenberg plugin, but it is not available in WordPress 5.4, which we still support. We consequently rely on a wrapper, `jetpackCreateInterpolateElement`.
+
+```jsx
+import Gridicon from 'components/gridicon';
+import { __ } from '@wordpress/i18n';
+import { jetpackCreateInterpolateElement } from 'components/create-interpolate-element';
+const getDocumentationLink = () => {
+	return jetpackjetpackCreateInterpolateElement(lement(
```

Comment:
Looks like bad copy/paste.

Reply:
Oops, my bad. Fixed in ee6cbb9ead498af704039be9f51cc3096de4ab26